packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-alternate-flows/main/java/getuserprofile.md

```diff
@@ -0,0 +1,29 @@
+
+Using the `tokenResponse` obtained from `IDXAuthenticationWrapper`'s `authenticate`
```

Comment:
Nit: `tokenResponse` -> `TokenResponse`

Reply:
Fixed in new branch vn-OKTA-408182-oie-java-embedded-app-2.